core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -3813,6 +3813,9 @@ components:
           description: Whether or not to delete intermediate output files when the
             workflow completes. See [Cromwell docs](https://cromwell.readthedocs.io/en/stable/wf_options/Google#google-pipelines-api-workflow-options)
             for more information.
+        executionPath:
+          type: string
+          description: The full GCS path indicating where execution took place.
```

Comment:
Nit: It's really more of a "root" than a "path". Actual executions will use paths that are generated by appending to this stub.

Reply:
Discussed via Slack- gonna change the terminology to be `submissionRoot` throughout